packages/x-tailwindcss/src/x-tailwind-plugin/components/button-group/default.ts

```diff
@@ -0,0 +1,31 @@
+import { TailwindHelpers } from '../../../types';
+
+/**
+ * Returns the default styles for component `button-group`.
+ *
+ * @param helpers - The {@link TailwindHelpers} to generate CSS.
+ * @returns The {@link CssStyleOptions} for the component.
+ */
+export function buttonGroupDefault(helpers: TailwindHelpers) {
+  const { theme } = helpers;
+
+  return {
+    display: 'inline-flex',
+    flexFlow: 'row nowrap',
+    color: theme('colors.neutral.50'),
+
+    '&:not([class*="gap"]) > .button + .button': {
```

Comment:
I don't get why we need the `:not([class*="gap"])`

Reply:
I will leave a comment in the code explaining that.
We remove the left border of the buttons to avoid double borders, which is more evident with the `outlined` variant.
If we apply a `padding` to the `button-group`, we should keep the left border because the buttons will be separated.